ONSdigital/dp-dd-api
# feature/initial-build-pipeline

### What

* Added build and deployment gubbins
* Added the `sbt-assembly` plugin for creating fat jars

### How to review

* Check the build for this branch is passing
* Check the application is deployed and configured correctly

### Who can review

Anyone but myself.
